### PR TITLE
Feat: Set up a local instance of Aurora using the CLI

### PR DESCRIPTION
**Pull request by birchmd**
The main goal of this PR is to introduce functionality to the CLI where it can be used to initialize a new Engine instance. This could be used on any network, but in particular the idea is that developers would use this to facilitate setting up a local Aurora Engine for interactive testing (this is as opposed to [Aurora Workspaces](https://github.com/aurora-is-near/aurora-workspace/), which is much better for automated/programmatic testing in a local environment).

This PR includes a new `write` command called `engine-init` for deploying and initializing a new Engine instance. It also includes a new class of commands called `init` which is meant to initialize the local environment itself; it includes subcommands to modify the local nearcore genesis file and the CLI's config file to work with the local nearcore node.

Additionally, this PR includes documentation describing how to use the CLI to set up a localnet. Reviewers may wish to start their review with this (`docs/localnet.md`) to see how the code changes manifest on the user level.

Finally, this PR includes some small changes to the output of the `deploy-code` and `solidity` subcommands. These changes are simply quality of life improvements to make the example usage at the end of the localnet documentation look nicer. We should probably review the output of other commands in the CLI to see if it can be made more useful, but that is out of scope for this PR.

### CODE REVIEW

File: src/cli/near.rs [hunk 1]
```diff
@@ -119,6 +132,36 @@ pub enum ReadCommand {
...
 pub enum WriteCommand {
+    /// Deploy and initialize a new instance of the Aurora Engine.
+    /// Uses the `engine_account_id` from the config as the target account.
+    /// `config.near_key_path` must point to a full access key for `engine_account_id`.
+    EngineInit {
+        /// Path to the Wasm artifact for the engine contract.
+        #[clap(short, long)]
+        wasm_path: String,
+        #[clap(short, long)]
+        chain_id: u64,
```
Comment (joshuajbouw) [inline src/cli/near.rs:143]: We should have documents for each of these, even if its self explanatory.

**Review by joshuajbouw — CHANGES_REQUESTED** [review]
Just a small change, mostly because this has to do with how `clap` gives information of arguments of the CLI. Overall, looks really good.

File: src/cli/near.rs (hunk 1 above)
Comment (birchmd) [reply src/cli/near.rs]: Additional documentation added in https://github.com/aurora-is-near/aurora-cli-rs/pull/8/commits/0765289b1f9de35ce6a6d1ef9e4228f9de38459d

**Comment by aleksuss** [comment]
Good work. Thank you. But it seems that we've done almost the same job. Now I need to merge these changes into #9, which would be challenging.

**Comment by birchmd** [comment]
@aleksuss I suggest splitting #9 into multiple smaller PRs, that should make resolving the conflicts much easier. Maybe something like

1. Use `anyhow` for errors
2. Update dependencies + rust version
3. Introduce simple+advanced distinction

**Review by mrLSD — APPROVED** [review]

**Pull request merged**